cinema/models.py

```diff
@@ -0,0 +1,17 @@
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    duration = models.IntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(300)]
+    )
+
+    class Meta:
+        verbose_name = "movies"
+
+    def __str__(self):
+        return self.title
+    
```

Comment:
please re-read the checklist

Reply:
I re-read checklist and didn't get what is the problem with this